tracing-test/src/lib.rs

```diff
@@ -0,0 +1,66 @@
+//! Testing utilities for `tracing`
+//!
+//! # Overview
+//!
+//! `tracing-test` offers mock implementations of useful `tracing` types to enable easier unit
+//! testing of `tracing`-based crates.
+//!
+//! # Usage
+//!
+//! Add the following to your `Cargo.toml`:
+//! ```toml
+//! tracing-test = "0.1"
+//! ```
+//!
+//! ## Mock subscriber
+//!
+//! ```rust
+//! # fn docs() {
+//! // TODO
+//! # }
+//! ```
```

Comment:
Can you add an end-to-end example of something like the `event_with_message` test in `tracing`?

Reply:
Done. I added a span to show the enter/exit/drop sequence, but I'm not sure if it's useful enough to justify the added lines. What do you think?